### PR TITLE
Added docs for service with SessionAffinity in WSL

### LINKED ISSUE #1740: [WSL2] Sync failed errors in kube-proxy for Service with SessionAffinity: ClientIP
**What happened**:
*iptables* fail to be updated on the nodes after a `Service` with `sessionAffinity: ClientIP` is created.
The issue manifests in requests beeing dropped to any Services that were created after the Service with session affinity.

kube-proxy pod is logging the following error:
```
E0720 14:29:10.934607       1 proxier.go:1507] Failed to execute iptables-restore: exit status 2 (iptables-restore v1.8.3 (legacy): Couldn't load match `recent':No such file or directory

Error occurred at line: 96
Try `iptables-restore -h' or 'iptables-restore --help' for more information.
)
I0720 14:29:10.934636       1 proxier.go:779] Sync failed; retrying in 30s
```

**What you expected to happen**:
*iptables* to be updated correctly so that requests could be routed to any Service in the cluster.

**How to reproduce it (as minimally and precisely as possible)**:
Create a `Service` with `sessionAffinity: ClientIP`
```yaml
apiVersion: v1
kind: Service
metadata:
  labels:
    alertmanager: main
  name: alertmanager-main
  namespace: monitoring
spec:
  ports:
  - name: web
    port: 9093
    targetPort: web
  selector:
    alertmanager: main
    app: alertmanager
  sessionAffinity: ClientIP
```

**Anything else we need to know?**:
Issue is reproducible with both `kubeProxyMode: iptables` (default) and `kubeProxyMode: ipvs`

**Environment:**

- kind version:
  - `kind v0.8.1 go1.13.8 linux/amd64`
  - `kind v0.9.0-alpha+95753c11434213 go1.15beta1 linux/amd64`
- Kubernetes version:
  - `Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.5", GitCommit:"e0fccafd69541e3750d460ba0f9743b90336f24f", GitTreeState:"clean", BuildDate:"2020-05-01T02:11:15Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}`
  - also tried `v1.18.2` (default with `kind v0.8.1`) and `v1.18.6` (default with `kind v0.9.0-alpha`)
- Docker version: Docker Desktop with WSL2
```yaml
Client:
 Debug Mode: false

Server:
 Containers: 1
  Running: 1
  Paused: 0
  Stopped: 0
 Images: 1
 Server Version: 19.03.8
 Storage Driver: overlay2
  Backing Filesystem: <unknown>
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
 init version: fec3683
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 4.19.104-microsoft-standard
 Operating System: Docker Desktop
 OSType: linux
 Architecture: x86_64
 CPUs: 24
 Total Memory: 25GiB
 Name: docker-desktop
 ID: D4I2:L4Y5:PGPS:CEUY:H3TU:C33L:HASQ:VZKB:53SE:SHQG:OOQV:BZMQ
 Docker Root Dir: /var/lib/docker
 Debug Mode: true
  File Descriptors: 48
  Goroutines: 57
  System Time: 2020-07-20T15:35:34.5632783Z
  EventsListeners: 3
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
 Product License: Community Engine

WARNING: bridge-nf-call-iptables is disabled
WARNING: bridge-nf-call-ip6tables is disabled
```
- OS: Windows 10 `(Build: 19041.388)`

### PR DESCRIPTION
**Pull request by anyname2**
Added documentation section about using `sessionAffinity: ClientIP` services in WSL2.

Resolves #1740

### CODE REVIEW

**Comment by aojea** [comment]
/ok-to-test
Have you opened a bug in wsl to include this module by default?

**Comment by anyname2** [comment]
I did now: https://github.com/microsoft/WSL/issues/7124

File: site/content/docs/user/using-wsl2.md [hunk 1]
```diff
@@ -84,6 +84,47 @@ nodes:
 1. create service `kubectl create service nodeport nginx --tcp=80:80 --node-port=30000`
 1. access service `curl localhost:30000`
 
+## Kubernetes Service with Session Affinity
```
Comment (aojea) [inline site/content/docs/user/using-wsl2.md:87]: @BenTheElder should we start a "Known issues" section here, and nest this inside it?

File: site/content/docs/user/using-wsl2.md [hunk 2]
```diff
@@ -84,6 +84,47 @@ nodes:
...
+apt update
+apt install -y git build-essential flex bison libssl-dev libelf-dev bc
+
+# Checkout WSL2 Kernel repo
+mkdir src
+cd src
+git init
+git remote add origin https://github.com/microsoft/WSL2-Linux-Kernel.git
+git config --local gc.auto 0
+git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +${WSL_COMMIT_REF}:refs/emotes/origin/build/linux-msft-wsl-5.4.y
```
Comment (tallaxes) [inline site/content/docs/user/using-wsl2.md:108]: `refs/emotes` looks like a typo ...

File: site/content/docs/user/using-wsl2.md (hunk 1 above)
Comment (BenTheElder) [reply site/content/docs/user/using-wsl2.md:87]: This is fine for now, we can change it later, I'm happy to have something covering this, thank you!

**Review by BenTheElder — APPROVED** [review]
/lgtm
/approve
/hold

**Comment by BenTheElder** [comment]
Thanks!

**Pull request merged**